index/src/naf.ts

```diff
@@ -1,9 +1,7 @@
 import allCodesNaf from "@socialgouv/codes-naf";
 
-const formatCodeNaf = (code: string) => code.replace(".", "") + "Z";
-
 const codesEntries: [string, string][] = allCodesNaf.map(({ id, label }) => [
-  formatCodeNaf(id),
+  id,
```

Comment:
should we use the initial, not altered code as given from `activitePrincipaleEtablissement` and `activitePrincipaleUniteLegale` ?

Reply:
From what I see, @socialgouv/code-naf don't have any letter at the end, where the `activitePrincipaleEtablissement` sometimes have a ending `z` but some other letter to